### PR TITLE
Uml 1058 - welsh emails

### PR DESCRIPTION
**Pull request by GemTay**
# Purpose

Sends emails in welsh if locale is set to cy

Fixes UML-1058

## Approach

Injected locale into EmailClient constructor and used if statements within each send email function to send the email in english or welsh

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [x] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

**Review by Raffers — DISMISSED** [review]

File: service-front/app/src/Actor/src/Handler/CreateAccountHandler.php [hunk 1]
```diff
@@ -93,13 +93,7 @@ public function handle(ServerRequestInterface $request): ResponseInterface
...
                 } catch (ApiException $ex) {
                     if ($ex->getCode() == StatusCodeInterface::STATUS_CONFLICT) {
-                        if ($ex->getMessage() === 'Another user has requested to change their email to ' . $emailAddress){
-                            // send email to inform the user who has not completed email reset saying
-                            // that someone has tried to use their email in account creation
-                            $this->emailClient->sendSomeoneTriedToUseYourEmailInEmailAccountCreation($emailAddress);
-                        } else {
-                            $this->emailClient->sendAlreadyRegisteredEmail($emailAddress);
-                        }
+                        $this->emailClient->sendAlreadyRegisteredEmail($emailAddress);
```
Comment (cooperaj) [inline service-front/app/src/Actor/src/Handler/CreateAccountHandler.php:96]: This is a change in functionality. Intended?

Comment (GemTay) [reply service-front/app/src/Actor/src/Handler/CreateAccountHandler.php:96]: Yeah if you look at the template ID's for `TEMPLATE_ID_ACCOUNT_CREATION_EMAIL_CHANGE_REQUESTED` and `TEMPLATE_ID_EMAIL_ADDRESS_ALREADY_REGISTERED` they were the same, so the if else statement was serving no purpose

**Review by williamfalconeruk — APPROVED** [review]
LGTM

**Review by Raffers — APPROVED** [review]

File: service-front/app/src/Actor/src/Handler/CreateAccountHandler.php (hunk 1 above)
Comment (cooperaj) [reply service-front/app/src/Actor/src/Handler/CreateAccountHandler.php:96]: 👍

**Pull request merged**